library/func_misc.py

```diff
@@ -487,7 +494,7 @@ def Call(self, args):
         name = args.PosStr()
         args.Done()
         return expr_eval.LookupVar(self.mem, name, scope_e.Dynamic,
-                                   loc.Missing)
+                                   args.BlamePos())
```

Comment:
For cases where BlamePos() is called on the non-error path, I wonder if we can just have a variant:

```
loc =
    ...
  | ArgList %ArgList
```

and then pass `args`, so it's essentially free

It may make the error message worse, but I feel like we could start with that, and then make the errors more precise when we encounter them

It seems like 90% of cases are only computing positions in error cases, which is good